### PR TITLE
[pose-detection] Add correctness test.

### PR DESCRIPTION
**Pull request by lina128**
This PR has below changes:
1. Set up browser test infra, see karma.conf.js.
2. Set up browserstack to test in cloud, see cloudbuild.yml. Use Chrome browser for now, will add more devices (ios and android) and browsers (firefox, safari, IE) to Nightly test in follow-up PR. In this PR, use WebGL backend, will add WASM backend in follow-up PR.
3. Add correctness test with static image. Will add video test in follow-up PR.
4. Change Blazepose return result to return scaled keypoint coordinates instead of normalized coordinates.
5. Change Demo to not special case Blazepose for scaling because it is now taken care of by the pipeline.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/638)

### CODE REVIEW

**Review by mhays-google — APPROVED** [review]
> Reviewed 13 of 13 files at r1.
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/638)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @bazarevsky, @ivan-grishchenko, and @pyu10055)

**Review by pyu10055 — APPROVED** [review]
Very nice, thank you!

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-:-MXm_uuf7r5tUguTxp8M:b-ltfquq)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @bazarevsky, @ivan-grishchenko, and @lina128)

___
*[pose-detection/package.json, line 48 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-MXmYxcxBpl90ZFBo0TY:-MXmYxcy5M-5J97hBPcm:b-l632as) ([raw file](https://github.com/tensorflow/tfjs-models/blob/a67a348864601610f3615179d2c3ba48b7b9dad5/pose-detection/package.json#L48)):*
> ```JSON
>     "tslint-no-circular-imports": "~0.7.0",
>     "typescript": "~3.5.3",
>     "yalc": "~1.0.0-pre.21"
> ```

update the yalc version to avoid security issues.
___
*[pose-detection/src/setup_test.ts, line 30 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-MXmZP6G5W0Zsrwn29p6:-MXmZP6G5W0Zsrwn29p7:bebuint) ([raw file](https://github.com/tensorflow/tfjs-models/blob/a67a348864601610f3615179d2c3ba48b7b9dad5/pose-detection/src/setup_test.ts#L30)):*
> ```XML
>   flags: {
>     'WEBGL_VERSION': 2,
>     'WEBGL_CPU_FORWARD': false,
> ```

cpu forward should be enabled
___
*[pose-detection/src/blazepose/blazepose_test.ts, line 108 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-MXm_8E9D-3SmgtK41J1:-MXm_8E9D-3SmgtK41J2:b55jmec) ([raw file](https://github.com/tensorflow/tfjs-models/blob/a67a348864601610f3615179d2c3ba48b7b9dad5/pose-detection/src/blazepose/blazepose_test.ts#L108)):*
> ```XML
>   MODEL_TYPE.forEach(type => {
>     it('test.', async () => {
>       const upperBodyOnly = type === 'upperbody' ? true : false;
> ```

avoid type as variable, used modelType instead.

File: pose-detection/src/blazepose/detector.ts [hunk 1]
```diff
@@ -181,7 +182,10 @@ export class BlazeposeDetector extends BasePoseDetector {
 
     image3d.dispose();
 
-    const pose: Pose = {score: poseScore, keypoints: actualLandmarksFiltered};
+    const keypoints = actualLandmarksFiltered != null ?
+        normalizedLandmarksToLandmarks(actualLandmarksFiltered, imageSize) :
```
Comment (ivan-grishchenko) [inline pose-detection/src/blazepose/detector.ts:186]: Looks like you didn't normalize before. Why it didn't affect the demo?

**Review by lina128 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/638)* status: :shipit: complete! 2 of 1 approvals obtained (waiting on @bazarevsky, @ivan-grishchenko, @lina128, and @pyu10055)

___
*[pose-detection/src/setup_test.ts, line 30 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-MXmZP6G5W0Zsrwn29p6:-MXmcTqZCEIdoktUfE1W:b-wehhhe) ([raw file](https://github.com/tensorflow/tfjs-models/blob/a67a348864601610f3615179d2c3ba48b7b9dad5/pose-detection/src/setup_test.ts#L30)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

cpu forward should be enabled
</blockquote></details>

For testing purpose, should we test both path?
___
*[pose-detection/src/blazepose/blazepose_test.ts, line 108 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-MXm_8E9D-3SmgtK41J1:-MXmcBRO48fgMs_h2kOR:bt6ybxa) ([raw file](https://github.com/tensorflow/tfjs-models/blob/a67a348864601610f3615179d2c3ba48b7b9dad5/pose-detection/src/blazepose/blazepose_test.ts#L108)):*
<details><summary><i>Previously, pyu10055 (Ping Yu) wrote…</i></summary><blockquote>

avoid type as variable, used modelType instead.
</blockquote></details>

There's no modelType, just a boolean flag, should I create a flag list instead? `UPPERBODY_ONLY = [true, false]`, and then iterate on this.
___
*[pose-detection/src/blazepose/detector.ts, line 186 at r2](https://reviewable.io/reviews/tensorflow/tfjs-models/638#-MXjC36U7VM2I9ajPwoq-r2-186:-MXmc7vF0rJIS7B1HEmy:b-6zv00o) ([raw file](https://github.com/tensorflow/tfjs-models/blob/a67a348864601610f3615179d2c3ba48b7b9dad5/pose-detection/src/blazepose/detector.ts#L186)):*
<details><summary><i>Previously, ivan-grishchenko wrote…</i></summary><blockquote>

Looks like you didn't normalize before. Why it didn't affect the demo?
</blockquote></details>

The demo take care the scaling before.

**Review by ivan-grishchenko — APPROVED** [review]

**Review by bazarevsky — APPROVED** [review]

**Pull request merged**